modules/mssqlserver/README.md

```diff
@@ -0,0 +1,52 @@
+# TestContainers MS SQL Server Module
+
+Testcontainers module for the MS SQL Server database.
+
+## Usage example
+
+Running MS SQL Server as a stand-in for in a test:
+
+```java
+public class SomeTest {
+
+    @Rule
+    public MSSQLServerContainer mssqlserver = new MSSQLServerContainer();
+    
+    @Test
+    public void someTestMethod() {
+        String url = mssqlserver.getJdbcUrl();
+
+        ... create a connection and run test as normal
+```
+
+> *Note:* Due to licencing restrictions you are required to accept an EULA for this container image. To indicate that you accept the MS SQL Server image EULA, Please place a file at the root of the classpath named `container-license-acceptance.txt`, e.g. at `src/test/resources/container-license-acceptance.txt`. This file should contain the line: `microsoft/mssql-server-linux:latest`
```

Comment:
@stefanhufschmidt I hope this is OK with you: we felt uncomfortable with setting the EULA acceptance env var automatically, so we've created this mechanism to allow devs to indicate their acceptance.

Reply:
That's right I've noticed this as well but I had not such a nice idea with this license file. I think it is a  nice solution! Well done.

Reply:
We might throw an Exception in this module if the EULA is not accepted here.

Reply:
Throwing the exception is already implemented.